README.rst

```diff
@@ -39,7 +39,8 @@ Then get the frontend building::
    $ yarn install
    $ yarn serve
 
-Then, you can visit the local site at ``http://localhost:8000/``.
```

Comment:
If you feel comfortable with `git rebase`, it's probably worth dropping this commit and the preceding commit from this branch outright. If not, I wouldn't sweat it too hard - low-noise Git history is nice, but not a must-have, and it's clear what this commit is doing.